frontend/redux/actions.js

```diff
@@ -57,6 +57,30 @@ export const requestAttributesFailed = (error, retryAction) => ({
   error: true,
 });
 
+/** @prop {string} Action to trigger a request for face data */
+export const REQUEST_OVERALL_STATS = 'REQUEST_OVERALL_STATS';
+export const requestOverallStats = () => ({
+  type: REQUEST_OVERALL_STATS,
+});
+
+/** @prop {string} Action to populate the annotations within the store when the request succeeds */
+export const RECEIVE_OVERALL_STATS = 'RECEIVE_OVERALL_STATS';
+export const receiveOverallStats = stats => ({
+  type: RECEIVE_OVERALL_STATS,
+  payload: stats,
+});
+
+/** @prop {string} Action to signal a data request failed */
+export const REQUEST_OVERALL_STATS_FAILED = 'REQUEST_OVERALL_STATS_FAILED';
+export const requestOverallStatsFailed = (error, retryAction) => ({
+  type: REQUEST_OVERALL_STATS_FAILED,
```

Comment:
Not necessary for this PR but this type could have additional error text added for it [in the content service](https://github.com/bocoup/image-annotator/blob/eb85b4cb4dec959f060247c821e58414f24d7add/frontend/services/content.js#L10)